sc-memory/sc-memory/scs/scs.g4

```diff
@@ -163,9 +163,9 @@ idtf_lvl1 returns [ElementHandle handle]
   ;
 
 idtf_edge returns [ElementHandle handle]
-  : '(' src=idtf_system
+  : '(' src=idtf_common
         c=connector attrs=attr_list?
-        trg=idtf_system
+        trg=idtf_common
```

Comment:
I'm not sure, that this is a correct fix. It can break parser. We should to discuss that